src/test/java/com/sap/cloudfoundry/client/facade/ApplicationsCloudControllerClientIntegrationTest.java

```diff
@@ -340,6 +345,45 @@ void updateApplicationMetadata() {
         }
     }
 
+    @Test
+    @DisplayName("Create a user provided service and verify its parameters")
+    void createUserProvidedServiceTest() {
+        String serviceName = "test-service-1";
+        try {
+            client.createUserProvidedServiceInstance(buildUserProvidedService(serviceName), USER_SERVICE_CREDENTIALS, SYSLOG_DRAIN_URL);
+            CloudServiceInstance service = client.getServiceInstance(serviceName);
+            assertEquals(SYSLOG_DRAIN_URL, service.getSyslogDrainUrl());
+            assertEquals(USER_SERVICE_CREDENTIALS, service.getCredentials());
+            assertTrue(service.isUserProvided());
+        } finally {
+            client.deleteServiceInstance(serviceName);
+        }
+    }
+
+    @Test
+    @DisplayName("Create a user provided service and update its parameters")
+    void updateUserProvidedServiceTest() {
+        String serviceName = "test-service-2";
+        Map<String, Object> updatedServiceCredentials = Map.of("newTestCredentialsKey", "newTestCredentialsValue");
+        String updatedSyslogDrainUrl = "newSyslogDrain";
+        try {
+            client.createUserProvidedServiceInstance(buildUserProvidedService(serviceName), USER_SERVICE_CREDENTIALS, SYSLOG_DRAIN_URL);
+            client.updateServiceParameters(serviceName, updatedServiceCredentials, updatedSyslogDrainUrl);
+            CloudServiceInstance service = client.getServiceInstance(serviceName);
+            assertEquals(updatedSyslogDrainUrl, service.getSyslogDrainUrl());
+            assertEquals(updatedServiceCredentials, service.getCredentials());
+        } finally {
+            client.deleteServiceInstance(serviceName);
+        }
+    }
+
+    private CloudServiceInstance buildUserProvidedService(String serviceName) {
+        return ImmutableCloudServiceInstance.builder()
+                                            .name(serviceName)
+                                            .type(ServiceInstanceType.USER_PROVIDED)
+                                            .build();
+    }
+
```

Comment:
I'd suggest creating a new class for these tests since they are related to the services rather than applications. 
ServicesCloudControllerClientIntegrationTest or something similar